client/my-sites/hosting/controller.js

```diff
@@ -6,20 +6,21 @@
 import React from 'react';
 import config from 'config';
 import page from 'page';
+import { get } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import Hosting from './main';
-import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { isBusinessPlan } from 'lib/plans';
```

Comment:
Note for self - I would still like to [move this logic up into the API](https://github.com/Automattic/wp-calypso/pull/36366#discussion_r328806040) once the majority of it is merged in/concrete.

Reply:
💯